worker/docker_client.py

```diff
@@ -224,19 +224,10 @@ def start_container(self, bundle_path, uuid, command, docker_image,
         # Set up the command.
         docker_bundle_path = '/' + uuid
         docker_commands = [
-            'BASHRC=$(pwd)/.bashrc',
-            # We pass several commands for bash to execute as a single
-            # argument (i.e. all commands appear in quotes with no spaces
-            # outside the quotes). The first commands appear in double quotes
-            # since we want environment variables to be expanded. The last
-            # appears in single quotes since we do not. The expansion there,
-            # if any, should happen when bash executes it. Note, since the
-            # user's command can have single quotes we need to escape them.
-            'bash -c '
-            + '"[ -e $BASHRC ] && . $BASHRC; "'
-            + '"cd %s; "' % docker_bundle_path
-            + '"export HOME=%s; "' % docker_bundle_path
-            + '\'(%s) >stdout 2>stderr\'' % command.replace('\'', '\'"\'"\''),
+            # TODO: Remove when codalab/torch uses ENV directives
```

Comment:
Yikes good to know. I just opened up the codalab/ubuntu:1.9 image and the bashrc file is also stored at ` /user/.bashrc`. 

I did actually test on this image before pushing, but I would need to devise more substantial tests (e.g. programs that use Scala) in order to test whether or not the environment variables work..